docs/cql/functions.rst

```diff
@@ -227,6 +227,32 @@ A number of functions are provided to “convert” the native types into binary
 takes a 64-bit ``blob`` argument and converts it to a ``bigint`` value. For example, ``bigintAsBlob(3)`` is
 ``0x0000000000000003`` and ``blobAsBigint(0x0000000000000003)`` is ``3``.
 
+
+Vector similarity functions :label-caution:`Experimental` :label-note:`ScyllaDB Cloud`
```

Comment:
The functions are available everywhere (though not very useful without the search component).